systems/controllers/CMakeLists.txt

```diff
@@ -16,6 +19,7 @@ pods_find_pkg_config(gurobi)
 if (gurobi_FOUND)
   add_library(drakeQPCommon SHARED QPCommon.cpp)
   target_link_libraries(drakeQPCommon drakeControlUtil drakeQP drakeTrajectories drakeLCMUtil)
+  pods_use_pkg_config_packages(drakeQPCommon bot2-lcmgl-client lcm) # debug
```

Comment:
do we want to keep this dependency on bot-lcmgl? (and do we want to keep the lcmgl drawing code in the controller?)